src/Spago/CLI.hs

```diff
@@ -193,16 +193,19 @@ parser = do
       , Repl <$> replPackageNames <*> sourcePaths <*> pursArgs <*> depsOnly
       )
 
+    firstNonEmpty x y = if null x then y else x
+    execArgs = firstNonEmpty <$> backendArgs <*> nodeArgs
```

Comment:
Why do we have to pick one? I'd expect we could just merge them?

Reply:
I have no strong preference. I had it as merge first, but thought that would more surprising than preferring one or the other. For instance, if one script written by one person uses node-args, and another adds backend-args on top, you might expect it to override the first one, since you'd be treating them as equivalent. But, if the semantics of repeating node-args is to combine them, then having both would imply combining for the same reason.